README.md

```diff
@@ -120,20 +122,113 @@ You can use Safari or Google Chrome as your default browser. If you run into a b
 [Follow the Node.js package manager install instructions](https://nodejs.org/en/download/package-manager/) for your distribution to ensure you are getting the latest version of Node.js. Many distributions only maintain an older version in their default repositories. Python 3 is only required for running machine learning (TensorFlow).
 
 ```sh
-curl -sL https://deb.nodesource.com/setup_16.x | sudo -E bash -
-sudo apt-get install -y nodejs npm git python3
+curl -sL https://deb.nodesource.com/setup_17.x | sudo -E bash - && sudo apt-get \
+install -y \
+nodejs npm git python3
 ```
 
-You may verify the installed versions with these commands:
+You may verify the installed versions with this command string:
 
 ```sh
-node -v
-npm -v
-git --version
+node \
+-v && npm \
+-v && git --version
 ```
 
 If you are running headless (i.e. as a server without a monitor attached) then you do not need to install a web browser and you can follow the tutorial for information on connecting remotely to the server.
 
+Alternatively, one could use [https://github.com/nymea/berrylan](https://github.com/nymea/berrylan) to setup a tool for using bluetooth to quickly assign WPA2 access on a WLAN on a Raspbian based Distro. Nymea also has tools for automation of IoT products to allow setting up SuperAlgos as a timed function without needing to learn how to code.
+
+Also, if you are having node version errors there is a chance you may need to read the information following this to install and use NVM to handle node versions. This is due to some distributions having out of date repositories in the package manager lists.
+
+#### Installing on Debian or Debian WSL/WSL2 
+(NVM & NPM Fix)
+
+For this to work you will need to [use NVM to install and control node] (https://github.com/nvm-sh/nvm)
+
+- You will need to remove any versions of node already installed on Debian due to the repositories currently being out of date.
+- - __This is necessary before proceeding.__ 
+
+```sh
+sudo apt \
+remove nodejs -y && \
+```
+
+```sh
+sudo apt \
+update && apt upgrade \
+-y sudo apt \
+install npm -y
+```
+
+```sh
+sudo apt \
+autoremove -y && \
+sudo apt autoclean -y
+```
+
+```sh
+sudo curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash
+```
```

Comment:
is there any reason why all of these consecutive code blocks are separated into different code blocks?

Reply:
To make it easier to read for new linux users. Nothing more.